### ISSUE TITLE
Combo (oversampd train_set + NON-oversampd val_set) = awful validation results

### ISSUE THREAD

**Issue by AlexPasqua** [AlexPasqua/DM-project#14]
Se faccio oversampling del training set, ma non del validation set, ottengo dei risultati orribili durante il training, anche se sul test set (con aggiunta di dati sintetici) le performance sono buone.

Il file è [Classification_NN_oversamp.ipynb sul branch class/NN/oversamp](https://github.com/AlexPasqua/DM-project/blob/class/NN/oversamp/Classification_NN_oversamp.ipynb)

Esempio di grafico delle performance del training:
![image](https://user-images.githubusercontent.com/33461106/100212708-fa93c280-2f0d-11eb-8b08-60f291503e89.png)
